bin/barclamp_lib.rb

```diff
@@ -466,6 +162,18 @@ def get_user_password
   end
 
   if key
+    if ::File.exists?(@crowbar_key_file) and ::File.readable?(@crowbar_key_file)
```

Comment:
Lint/DeprecatedClassMethods: `File.exists?` is deprecated in favor of `File.exist?`.<br>Style/AndOr: Use `&&` instead of `and`. (https://github.com/bbatsov/ruby-style-guide#no-and-or-or)